PhysicsTools/NanoAOD/interface/MatchingUtils.h

```diff
@@ -0,0 +1,54 @@
+#ifndef NanoAOD_MatchingUtils_h
+#define NanoAOD_MatchingUtils_h
+
+/*#include <utility>
+#include <vector>
+#include "CommonTools/Utils/interface/StringCutObjectSelector.h"
+*/
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant.
It seems as if you want to also support full FWK with an option.
If so, I suggest to add a compiler switch

Reply:
the functions are just from a set of python utility that we converted to c++
the commented code is an unfinished port from python and will be uncommented once finished. At the moment are not used but they (most likely) will in some of the next PRs